### PR TITLE
Share panel dashboard add images

### PR DESCRIPTION
**Pull request by achatterjee-grafana**
Added relevant images to the  "Share a panel" and "Share a dashboard" topics.

### CODE REVIEW

File: docs/sources/sharing/share-dashboard.md [hunk 1]
```diff
@@ -23,23 +25,25 @@ To share a direct link:
...
 provided by Grafana Labs that allows you to publish dashboard snapshots to an external Grafana instance. The same rules still apply: anyone with the link can view it. You can set an expiration time if you want the snapshot removed after a certain time period.
 
-{{< docs-imagebox img="/img/docs/v50/share_panel_modal.png" max-width="700px" >}}
+{{< docs-imagebox img="/img/docs/sharing/Share_Dashboard_Snapshot.png" max-width="700px" >}}
 
 To publish a snapshot:
 
 1. Click on **Local Snapshot** or **Publish to snapshot.raintank.io**. This generates the link of the snapshot.
-1. Copy the snapshot link, and share it either within your organization or publicly on the web.
+2. Copy the snapshot link, and share it either within your organization or publicly on the web.
```
Comment (oddlittlebird) [inline docs/sources/sharing/share-dashboard.md:41]:
```suggestion
1. Copy the snapshot link, and then share it either within your organization or publicly on the web.
```

File: docs/sources/sharing/share-panel.md [hunk 2]
```diff
@@ -14,6 +14,8 @@ To share a panel:
 1. Click a panel title to open the panel menu.
 1. Click **Share**. The share dialog opens and shows the Link tab.
 
+{{< docs-imagebox img="/img/docs/sharing/Share_Panel_Direct_Link.png" max-width="700px" >}}
```
Comment (oddlittlebird) [inline docs/sources/sharing/share-panel.md:17]: See earlier comments about image link and file name.

File: docs/sources/sharing/share-panel.md [hunk 3]
```diff
@@ -31,20 +33,22 @@ Here is an example of a link to a server-side rendered PNG:
...
+#### Query string parameters for server-side rendered images
 
 - **width:** width in pixels. Default is 800.
 - **height:** height in pixels. Default is 400.
 - **tz:** timezone in the format `UTC%2BHH%3AMM` where HH and MM are offset in hours and minutes after UTC
 - **timeout:** number of seconds. The timeout can be increased if the query for the panel needs more than the default 30 seconds.
 - **scale:** numeric value to configure device scale factor. Default is 1. Use a higher value to produce more detailed images (higher DPI). Supported in Grafana v7.0+.
 
-## Publish snapshots
+## Publish snapshot
```
Comment (oddlittlebird) [inline docs/sources/sharing/share-panel.md:44]:
```suggestion
## Publish a snapshot
```

File: docs/sources/sharing/share-panel.md [hunk 4]
```diff
@@ -31,20 +33,22 @@ Here is an example of a link to a server-side rendered PNG:
...
 
 A panel snapshot shares an interactive panel publicly. Grafana strips sensitive data leaving only the visible metric data and series names embedded into your dashboard. Panel snapshots can be accessed by anyone with the link.
 
 You can publish snapshots to your local instance or to [snapshot.raintank.io](http://snapshot.raintank.io). The latter is a free service provided by [Raintank](http://raintank.io), that allows you to publish dashboard snapshots to an external Grafana instance. You can optionally set an expiration time if you want the snapshot to be removed after a certain time period.
 
+{{< docs-imagebox img="/img/docs/sharing/Share_Panel_Snapshot.png" max-width="700px" >}}
```
Comment (oddlittlebird) [inline docs/sources/sharing/share-panel.md:50]: See earlier comments about image link and file name.

File: docs/sources/sharing/share-panel.md [hunk 5]
```diff
@@ -57,6 +61,8 @@ If you created a snapshot by mistake, click **delete snapshot** to remove the sn
 
 You can embed a panel using an iframe on another web site. Unless anonymous access permission is enabled, the viewer must be signed into Grafana to view the graph.
 
+{{< docs-imagebox img="/img/docs/sharing/Share_Panel_Embedded_Link.png" max-width="700px" >}}
```
Comment (oddlittlebird) [inline docs/sources/sharing/share-panel.md:64]: See earlier comments about image link and file name.

**Review by oddlittlebird — CHANGES_REQUESTED** [review]

File: docs/sources/sharing/share-panel.md (hunk 3 above)
Comment (achatterjee-grafana) [reply docs/sources/sharing/share-panel.md:44]: Done

File: docs/sources/sharing/share-dashboard.md (hunk 1 above)
Comment (achatterjee-grafana) [reply docs/sources/sharing/share-dashboard.md:41]: Updated numbering.

**Comment by achatterjee-grafana** [comment]
@oddlittlebird, I would like to keep the filenames as is and finish this task, so that I can move on. I have noted down the file naming convention you would like to use, and will use them in the future. I will also fix the file names here in a separate PR. Hope that is OK with you.

File: docs/sources/sharing/share-panel.md (hunk 5 above)
Comment (achatterjee-grafana) [reply docs/sources/sharing/share-panel.md:64]: Will fix in separate PR.

File: docs/sources/sharing/share-panel.md (hunk 4 above)
Comment (achatterjee-grafana) [reply docs/sources/sharing/share-panel.md:50]: Will fix in separate PR.

File: docs/sources/sharing/share-panel.md (hunk 2 above)
Comment (achatterjee-grafana) [reply docs/sources/sharing/share-panel.md:17]: Will fix in separate PR.

File: docs/sources/sharing/share-dashboard.md [hunk 6]
```diff
@@ -23,23 +25,25 @@ To share a direct link:
...
+## Publish snapshot
 
 A dashboard snapshot shares an interactive dashboard publicly. Grafana strips sensitive data like queries
 (metric, template and annotation) and panel links, leaving only the visible metric data and series names embedded into your dashboard. Dashboard snapshots can be accessed by anyone with the link.
 
 You can publish snapshots to your local instance or to [snapshot.raintank.io](http://snapshot.raintank.io). The latter is a free service
 provided by Grafana Labs that allows you to publish dashboard snapshots to an external Grafana instance. The same rules still apply: anyone with the link can view it. You can set an expiration time if you want the snapshot removed after a certain time period.
 
-{{< docs-imagebox img="/img/docs/v50/share_panel_modal.png" max-width="700px" >}}
+{{< docs-imagebox img="/img/docs/sharing/Share_Dashboard_Snapshot.png" max-width="700px" >}}
```
[the remark this answers is not in the document]
Comment (achatterjee-grafana) [reply docs/sources/sharing/share-dashboard.md:36]: Noted, will fix is separate PR.

File: docs/sources/sharing/share-dashboard.md [hunk 7]
```diff
@@ -14,6 +14,8 @@ To share a dashboard:
 1. Go to the home page of your Grafana instance.
 1. Click on the share icon in the top navigation. The share dialog opens and shows the Link tab.
 
+{{< docs-imagebox img="/img/docs/sharing/Share_Dashboard_Direct_Link.png" max-width="700px" >}}
```
[the remark this answers is not in the document]
Comment (achatterjee-grafana) [reply docs/sources/sharing/share-dashboard.md:17]: Noted.

**Comment by oddlittlebird** [comment]
The problem with not including the version number is that it raises the possibility of someone just replacing the image at a future date. We need to include the version number both so that people know which version the screenshot applies to and to protect image links in past documentation versions.

**Review by oddlittlebird — APPROVED** [review]

**Pull request merged**